Documentation/helm-operator.md

```diff
@@ -148,6 +154,9 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.volumeReplication.image`       | Volume Replication Controller image.                                                                                        | `quay.io/csiaddons/volumereplication-operator:v0.3.0`     |
 | `csi.csiAddons.enabled`     | Enable CSIAddons                                                                                                  | `false`                                                   |
 | `csi.csiAddons.image`       | CSIAddons Sidecar image.                                                                                        | `quay.io/csiaddons/k8s-sidecar:v0.2.1`     |
+| `csi.nfs.enabled`     | Enable nfs driver.                                                                                                  | `false`                                                   |
+| `csi.nfs.image`       | NFS nodeplugin image. image.                                                                                        | `mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0`     |
```

Comment:
Is it more clear to include "driver" in the name here? @Madhu-1 thoughts?

```suggestion
| `csi.nfsDriver.image`       | NFS nodeplugin image. image.                                                                                        | `mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0`     |
```

Reply:
I think just `nfs` is fine.